src/main.jsx

```diff
@@ -33,40 +26,7 @@ const router = createBrowserRouter([
   {
     path: "/",
     errorElement: <ErrorPage />,
-    children: [
-      {
-        path: "",
```

Comment:
Does it make more sense to extract the whole router into its own file?

Reply:
I also consider this, but that makes it harder to extract the path in the HomePage Component and also, I thought it would be easier to see in main.jsx still the router element